tests/test_models.py

```diff
@@ -0,0 +1,15 @@
+# ##
+# generic tests for model plugins
+# ##
+
+import pytest
+
+import brainscore_vision
+
+
+@pytest.mark.travis_slow
+@pytest.mark.parametrize('identifier',
```

Comment:
@mschrimpf - if we want to ensure that we can at minimum load all models (and maybe not a bad idea for other plugins as well) do we want to parametrize this with a list of ALL identifiers that we extract from the registry builder?

Reply:
I added `alexnet` in https://github.com/brain-score/brain-score/pull/408/commits/cce169123c6dbdeaf313116a76de09da26127c90. I think it is a reasonable standard to have all models in the registry tested here.